ui/src/components/Publication/RelatedPublications/index.tsx

```diff
@@ -12,7 +12,15 @@ const RelatedPublications: React.FC<Props> = (props) => {
     const { recent, relevant } = props.crosslinks.data;
     const totalCrosslinks = props.crosslinks.metadata.total;
     const [modalVisibility, setModalVisibility] = useState(false);
+    const [modalKey, setModalKey] = useState(0);
     const showShowAllButton = totalCrosslinks > 5;
+
+    const openModal = () => {
+        setModalVisibility(true);
+        // Resets modal state
+        setModalKey(modalKey + 1);
```

Comment:
Like this, was going to suggest something similar for the other PR but it would have required too much refactoring